core/domain/prod_validators_test.py

```diff
@@ -451,10 +446,11 @@ def test_maximum_of_ten_errors_are_emitted(self):
                     'property_name': 'title',
                     'new_value': 'New title %s' % i
                 })], 'Changes.')
+        self.process_and_flush_pending_tasks()
+
         exp_models.ExplorationModel.get_by_id('0').delete(
             self.user_id, '', [])
 
-        self.process_and_flush_pending_tasks()
```

Comment:
Why this change?

Reply:
The change was made after updating local branch(this was updated in master branch)

Reply:
Is this an expected change? That's what I wanted to check, if not then please undo these changes! :) [Ditto below]

Reply:
Yes, the changes are expected